test/grid.html

```diff
@@ -241,7 +241,57 @@
         });
       });
     });
- 
+
+    test('last visible items should change on resize', function(done) {
+      flush(function() {
+        // Physical scroll bar causes there to be less columns than expected.
+        // Force list to be wide enough to allow for 3 columns and the scroller.
+        list.style.width = '320px';
```

Comment:
This is a bit of a hack, but I wasn't sure how else to handle the issue. The initial layout assumes full width, but the later layout in response to the `iron-resize` uses full width minus one physical scroll bar, which breaks the anticipated layout. Not sure if I should do some deeper changes to `x-grid`, create another helper, just go with this approach being not a lot of tests use `iron-resize`, or whether this merits its own issue where the layout math is updated in `iron-list`.